x-pack/test/api_integration/apis/asset_manager/bootstrap_apm_synthtrace.ts

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { APM_TEST_PASSWORD } from '@kbn/apm-plugin/server/test_helpers/create_apm_users/authentication';
+import {
+  ApmSynthtraceEsClient,
+  ApmSynthtraceKibanaClient,
+  createLogger,
+  LogLevel,
+} from '@kbn/apm-synthtrace';
+import url from 'url';
+import { FtrProviderContext as InheritedFtrProviderContext } from '../../ftr_provider_context';
+
+export async function bootstrapApmSynthtrace(
+  context: InheritedFtrProviderContext,
+  kibanaClient: ApmSynthtraceKibanaClient
+) {
+  const es = context.getService('es');
+
+  const kibanaVersion = await kibanaClient.fetchLatestApmPackageVersion();
+  // Feels like this step takes a lot of time, is it needed?
+  // await kibanaClient.installApmPackage(kibanaVersion);
```

Comment:
I don't think this is needed for the data we produce now but we should consider installing `system` and `kubernetes` packages to setup the mappings

Reply:
Yeah, totally. We'll have to figure that out when it becomes needed.